src/linpred.jl

```diff
@@ -263,7 +263,7 @@ response(obj::LinPredModel) = obj.rr.y
 
 fitted(m::LinPredModel) = m.rr.mu
 predict(mm::LinPredModel) = fitted(mm)
-StatsModels.formula(obj::LinPredModel) = modelframe(obj).formula
+StatsModels.formula(::LinPredModel) = throw(ArgumentError("model was fitted without a formula"))
```

Comment:
There is no modelframe if it's not a `TableRegressionModel`. If it is a `TableRegressionModel`, there's already a formula method defined.